src/transformers/tokenization_utils_base.py

```diff
@@ -2070,7 +2074,7 @@ def from_pretrained(
                 "files are necessary for the tokenizer to operate."
             )
 
```

Comment:
also from gguf should never read any additional file so we should find a way to return early